src/shared/api/mock-factories/block.ts

```diff
@@ -1,12 +1,28 @@
-import { randNumber } from '@ngneat/falso';
+import { randNumber, randPastDate } from '@ngneat/falso';
 import { hash } from './utils';
+import { transactionList } from '../mock-factories/transaction';
 
 export function makeBlock(block_hash?: string): BlockShallow {
   return {
     block_hash: block_hash ?? hash(64),
     height: randNumber(),
     rejected_transactions: randNumber({ min: 0, max: 5 }),
-    timestamp: new Date(Date.now() - 10000).toISOString(),
+    timestamp: randPastDate().toISOString(),
     transactions: randNumber({ min: 1, max: 20 }),
   };
 }
+
+export function makeBlockDetails(height?: number): Block {
+  return {
+    block_hash: hash(64),
+    parent_block_hash: hash(64),
+    rejected_transactions_merkle_root_hash: hash(64),
+    transactions_merkle_root_hash: hash(64),
+    invalidated_blocks_hashes: [],
+    height: height ?? randNumber(),
+    rejected_transactions: [],
+    timestamp: randPastDate().toISOString(),
+    transactions: transactionList(400),
```

Comment:
Unfortunately, it won't work. The actual transactions from the backend will be string HEXes of serialized transactions. You should use `@iroha2/data-model` package in order to work with it.